ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml

```diff
@@ -205,13 +205,12 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       COMPUTE_NODE_TYPE: m6g.xlarge
       CONTROL_PLANE_INSTANCE_TYPE: m6g.2xlarge
-      E2E_RUN_TAGS: '@disconnected'
       OCP_ARCH: arm64
       TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-sno
-- as: aws-ipi-disc-priv-sno-amd-f28-destructive
+    workflow: cucushift-installer-rehearse-aws-ipi-sno-etcd-encryption
+- as: aws-ipi-disc-priv-sno-etcd-encryption-amd-f28-destructive
```

Comment:
I think we should do the same change as the above job.

Reply:
@jianlinliu You mean I should create one workflow `ref: etcd-encryption`?

Reply:
As a common ci job, we always create 2 jobs, one is for destructive, another for non-destructive testing, so if you changed the above non-destructive ci, accordingly the destructive ci job should have the same change. That means `aws-ipi-disc-priv-sno-etcd-encryption-amd-f28-destructive` should be changed to `aws-ipi-sno-etcd-encryption-amd-f28-destructive`

Reply:
Updated